src/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/.template.config/template.json

```diff
@@ -111,11 +119,35 @@
         {
           "choice": "Windows",
           "description": "Windows authentication"
+        },
+        {
+          "choice": "IndividualTokenService",
+          "description": "Individual authentication with a token server"
         }
       ],
       "defaultValue": "None",
       "description": "The type of authentication to use"
     },
+    "AspNetCoreVersion": {
+      "type": "parameter",
+      "datatype": "string",
+      "description": "The version for AspNet Core packages",
```

Comment:
Does this show up to users?

Reply:
Nope. They are hidden parameters on the CLI so they don't show in dotnet new.

Reply:
Ok. Should still fix to the proper product name `ASP.NET Core`.